lib/reporter.js

```diff
@@ -93,6 +93,7 @@ module.exports = async (data, templateFile, outputFile, showUnique) => {
         const report = await generateTemplate(modifiedData, templateFile);
 
         await writeReport(report, outputFile);
+        return Promise.resolve(modifiedData);
```

Comment:
Why we need `Promise.resolve`?
It is a async function and just plain return would do the trick. right?

Reply:
Ah you're correct, I'm still used to the `.then` callback-hell myself but with `async/await` this seems to be the case indeed.